packages/react-components/react-nav-preview/src/components/NavDrawer/renderNavDrawer.tsx

```diff
@@ -5,15 +5,14 @@ import { assertSlots } from '@fluentui/react-utilities';
 import type { NavDrawerState } from './NavDrawer.types';
 import { NavContextValues } from '../NavContext.types';
 import { NavProvider } from '../NavContext';
-import { InlineDrawerSlots } from '@fluentui/react-drawer';
+import { DrawerSlots } from '@fluentui/react-drawer';
 
 /**
  * Render the final JSX of NavDrawer
  */
 export const renderNavDrawer_unstable = (state: NavDrawerState, contextValues: NavContextValues) => {
-  assertSlots<InlineDrawerSlots>(state);
+  assertSlots<DrawerSlots>(state);
 
-  // TODO Add additional slots in the appropriate place
   return (
     <NavProvider value={contextValues.nav}>
       <state.root />
```

Comment:
We could remove the need for assert slots if we do this instead:
```ts
{renderDrawer(state as DrawerState)}
```

you should test this locally as well though

Reply:
changed, tested and pushed - good call!